example/src/main/java/com/github/appintro/example/ui/java/JavaIntro.java

```diff
@@ -0,0 +1,54 @@
+package com.github.appintro.example.ui.java;
+
+import android.os.Bundle;
+
+import androidx.annotation.Nullable;
+import androidx.fragment.app.Fragment;
+
+import com.github.appintro.AppIntro;
+import com.github.appintro.AppIntroFragment;
+import com.github.appintro.example.R;
+
+
+public class JavaIntro extends AppIntro {
```

Comment:
You need to add this class inside: https://github.com/AppIntro/AppIntro/blob/master/example/src/main/java/com/github/appintro/example/ui/IntroEntry.kt

Otherwise it won't show up in the menu in the sample app.